PWGHF/D2H/Tasks/taskCharmPolarisation.cxx

```diff
@@ -806,6 +814,8 @@ struct TaskPolarisationCharmHadrons {
               outputMl[2] = candidate.mlProbLcToPKPi()[2];
             }
           }
+          // invariant mass of the KPi pair
+          invMassKPiLc = RecoDecay::m(std::array{candidate.pVectorProng1(), candidate.pVectorProng2()}, std::array{massKaon, massPi});
```

Comment:
I would add a function in the `hfHelper` for the massKPi calculation, as for the massPKPi, similar to those for the massKK of the Ds https://github.com/AliceO2Group/O2Physics/blob/d3b76b374b0aa86092f1bf70adf77eecffbc7b63/PWGHF/Core/HfHelper.h#L245-L255